### PR TITLE
#316 Fix large stack of destinations

### LINKED ISSUE #316: Fix large stack of destinations
Pop up to the start destination of the graph to avoid building up a large stack of destinations on the back stack as users select items

### PR DESCRIPTION
**Pull request by michal-kucznerowicz**

### CODE REVIEW

**Review by kmaslowiec — APPROVED** [review]

**Comment by kmaslowiec** [comment]
https://developer.android.com/jetpack/compose/navigation#bottom-nav

File: app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt [hunk 1]
```diff
@@ -112,6 +113,7 @@ class MainActivity : ComponentActivity() {
 
 	private fun NavHostController.navigateWithSingleTop(item: NavigationItem) {
 		navigate(item.route) {
+			popUpTo(graph.findStartDestination().id)
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt:116]: @michal-kucznerowicz Could describe what kind of behavior does it fix? Mentioned issue has only the description copied from the documentation but I'm not sure if anyone knows how it changes the navigation.

Comment (michal-kucznerowicz) [reply app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt:116]: For example, let's go to PartiesScreen then CameraScreen then PartiesScreen then CameraScreen.
Before this change, the stack looked like: A -> B -> A -> B
After this change, the stack will look like: A -> B
It's just simplified. There is no large stack of destinations.

Comment (MarekMacko) [reply app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt:116]: Nice, thank you 🍻

**Pull request merged**